File: cpp/basics/files.cpp
// create and open text-file
ofstream file("test.txt");

// write to file
file << "This is a string.\n";

// read from file to variable
string text;
ifstream file("test.txt");

// read line
getline(file, text);
text
// This is a string.

// or
while (getline(file, text)) {
    cout << text << endl;
};
// This is a string.

// close file
file.close();

File: pascal/basics/files.pas
PROGRAM files;
VAR
    f: file of string;

    // variable for text
    text: string;
BEGIN
    // write to file
    ASSIGN(f, 'test.txt');
    REWRITE(f);
    WRITE(f, 'hello pascal');
    CLOSE(f);

    // read from file
    ASSIGN(f, 'test.txt');
    RESET(f);
    while not EOF(f) do
        BEGIN
            READ(f,text);
            WRITELN(text);
        END;
    // end while
    CLOSE(f);
    // hello pascal
END.